packages/neuron-wallet/src/services/wallets.ts

```diff
@@ -74,6 +74,12 @@ export default class WalletService {
     throw new Error('Failed to create wallet')
   }
 
+  public validate = ({ id, password }: { id: string; password: string }) => {
+    const wallet = walletStore.getWallet(id)
+    Key.checkPassword(wallet.keystore, password)
+    return Key.checkPassword(wallet.keystore, password)
```

Comment:
Instead of having a `Key.,checkPassword` static method, it's making more sense to have a `Keystore.checkPassword` instance method here.

Reply:
I watch pr of keystore. Finding `fromKey` missing in mock data.I confused with key and keystore in the past.I refactor this part now.In fact,`fromKeystore` don`t need password.`fromKey` does